### PR TITLE
GitHub Action

### PR DESCRIPTION
**Pull request by manics**
This adds the Dockerfile from https://github.com/ome/ome-ansible-molecule/pull/24 along with the `action.yml` file needed to turn this repo into a GitHub action. 

The intial action files were created by cherry-picking all commits from https://github.com/robertdebock/molecule-action/tree/79ad4ffcb2c2e97c2b451753e1bfc1c17473602c

https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/about-actions#creating-a-readme-file-for-your-action only mentions `README.md`, so there will be two README files, one for the Pypi package and one for the GitHub action.

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [ANSIBLE-ROLES-merge#1234](https://ci.openmicroscopy.org/job/ANSIBLE-ROLES-merge/1234/). See the [console output](https://ci.openmicroscopy.org/job/ANSIBLE-ROLES-merge/1234/consoleText) for more details.
Possible conflicts:
  - PR #24 manics 'Add Dockerfile for running molecule'
    - Dockerfile

--conflicts

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [ANSIBLE-ROLES-merge#1235](https://ci.openmicroscopy.org/job/ANSIBLE-ROLES-merge/1235/). See the [console output](https://ci.openmicroscopy.org/job/ANSIBLE-ROLES-merge/1235/consoleText) for more details.
Possible conflicts:
  - PR #24 manics 'Add Dockerfile for running molecule'
    - Dockerfile

--conflicts

File: action.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    description: The molecule scenario to run.
+    required: false
+    default: default
+
+runs:
+  using: docker
+  # It's possible to build the Docker image in the action:
+  # image: 'Dockerfile'
+  # but this takes around 2m20s compared to 20s to pull a pre-built image
+  image: 'docker://manics/docker-github-action-molecule:latest'
```
Comment (joshmoore) [inline action.yml:23]: What's the status of the build for this image?

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [ANSIBLE-ROLES-merge#1256](https://ci.openmicroscopy.org/job/ANSIBLE-ROLES-merge/1256/). See the [console output](https://ci.openmicroscopy.org/job/ANSIBLE-ROLES-merge/1256/consoleText) for more details.
Possible conflicts:
  - PR #24 manics 'Add Dockerfile for running molecule'
    - Dockerfile

--conflicts

**Comment by manics** [comment]
Replaced by https://github.com/ome/action-ome-ansible-molecule

**Pull request closed without merging**